README.md

```diff
@@ -40,6 +40,39 @@ Some choices for how to run services in your development environment:
     * PostgreSQL 9.5+
     * Mailcatcher 0.7.1
 
+## Setting up Ruby
+
+The easiest way to manage different installations of Ruby is with [rbenv](https://github.com/rbenv/rbenv). After following the installation instructions on the rbenv repo and restarting your terminal, you'll be able to run `rbenv install` in the root directory of the repository. rbenv will handle installing Ruby and switching to it for you when you enter the repo (so no need to worry about your other Ruby installs!) After that, run `gem install bundler` and you'll be all set to run `bundle install`.
+
+If you run into a problem where your bundle install says you're on an older version of bundler than what's in the lockfile, run `gem update --system` to update RubyGems, then `gem install bundler:1.17.3`
+
+Depending on your system, you might have trouble building the `pg` gem, probably due to a missing `libpq-fe.h`. To get the necessary libraries installed:
+* Debian/Ubuntu: `sudo apt-get install libpq-dev`
+* On Enterprise Linux (CentOS/RHEL/Fedora/Amazon Linux/Sci Linux): `yum install postgresql-devel`
+* For Mac (Homebrew): `brew install postgresql`
+
+## Setting up Node
+
+The easiest way to manage different installations with node.js is with [nvm](https://github.com/nvm-sh/nvm). After following the installation instructions, type `nvm install` to install and run the right version of node.js.
```

Comment:
`nodenv` also exists with the same syntax as `rbenv`, which is nice for consistency's sake. Do you think that's worth mentioning here? And if we do mention it, I'm not sure if it's better to fold this out into a separate readme with a link or to leave it here because it is the important info that people new to the project need to see. What do you think?